pkgs/top-level/all-packages.nix

```diff
@@ -9026,6 +9026,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreServices;
   };
 
+  mdbook-emoji = callPackage ../tools/text/mdbook-emoji { };
```

Comment:
```suggestion
  mdbook-emojicodes = callPackage ../tools/text/mdbook-emojicodes { };
```

Reply:
ah! thanks. Brought to you by Overshight.